src/utils.cpp

```diff
@@ -18,3 +18,14 @@ void EnsureTrailingSlash(QString* path) {
 QString FileNameFromPath(const QString& path) {
   return path.mid(path.lastIndexOf("/", -2) + 1);
 }
+
+ReconstructionStatus StatusFromInt(int status) {
```

Comment:
the method is redundant. I'm pretty sure we can explicitly convert int to enum member.
http://en.cppreference.com/w/cpp/language/enum